app/services/clients/stripe/stripe_client.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+// NPM dependencies
+const Stripe = require('stripe')
+const ProxyAgent = require('https-proxy-agent')
+
+// Local dependencies
+const StripeBankAccount = require('./stripeBankAccount.model')
+
+// Constants
+const STRIPE_LIVE_API_KEY = process.env.STRIPE_LIVE_API_KEY
+const STRIPE_TEST_API_KEY = process.env.STRIPE_TEST_API_KEY
+const STRIPE_HOST = process.env.STRIPE_HOST
+const STRIPE_PORT = process.env.STRIPE_PORT
+
+const getStripe = function getStripe (gatewayAccount) {
+  const apiKey = (gatewayAccount.type === 'test') ? STRIPE_TEST_API_KEY : STRIPE_LIVE_API_KEY
```

Comment:
I don't think it makes sense for this to have access to both the `TEST` and `LIVE` API keys at once -- it seems this process moves a gateway account from being `test` to `live` which isn't quite understood by this conditional (i.e there won't exist Stripe test accounts for a test Gateway Accounts)

Potentially we shouldn't include this dual logic until we have a clear use case (instead favouring just `STRIPE_API_KEY` being set by the environment with test or production keys as required).

Reply:
Agreed, we'll change this